docs/2.8.0/docs/daml-finance/instruments/generic.rst

```diff
@@ -159,4 +159,18 @@ Finally, it is possible to apply the Election according to the lifecycle rule pr
 This creates lifecycle effects, which can be claimed and settled in the usual way (as described in
 :doc:`Getting Started: Lifecycling <../tutorials/getting-started/lifecycling>`). However, the holding contract
 used to claim the effect must be compatible with the election that has been made: if Alice made an
-election and *electorIsOwner = True*, then only a holding where *owner = alice* will be accepted.
\ No newline at end of file
+election and *electorIsOwner = True*, then only a holding where *owner = alice* will be accepted.
+
+Frequently Asked Questions
+**************************
+
+When is a Generic Instrument preferred over a strongly typed one?
```

Comment:
i wouldn't use the term "strongly typed" instrument (I tried to remove it from the rest of the doc). This is because we never define it and it's slightly confusing as the generic instrument is also strongly typed.

What about 

 ```suggestion
When is a Generic Instrument preferred over a bond or swap?
```